specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/file.json

```diff
@@ -471,6 +452,13 @@
             "required": false,
             "type": "string",
             "description": "Optional, used to delete a snapshot."
+          },
+          {
+            "name": "$include",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Optional. Valid values are: snapshots, leased-snapshots, none. The default value is snapshots. For 'snapshots', the file share is deleted including all of its file share snapshots. If the file share contains leased-snapshots, the deletion fails. For 'leased-snapshots', the file share is deleted included all of its file share snapshots (leased/unleased). For 'none', the file share is deleted if it has no share snapshots. If the file share contains any snapshots (leased or unleased), the deletion fails."
```

Comment:
Please confirm the default value is snapshots on server.

Reply:
Confirmed